cli.md

```diff
@@ -27,9 +34,17 @@ These options influence how Guardian interacts with GitHub:
   This option can also be specified with the GITHUB_ACTIONS environment variable.
 * **-github-owner="organization-name"** - The GitHub repository owner.
 * **-github-repo="repository-name"** - The GitHub repository name.
-* **-github-token="string"** - The GitHub access token to make GitHub API calls. This
+* **-github-token="string"** - The GitHub access token used for Terraform to make GitHub API calls. This
```

Comment:
This is slightly true, if the guardian github token isnt supplied, this is used as a fallback IIRC. We shouldn't mention terraform here I dont think. Terraform handles its own auth its own way.